Gemfile

```diff
@@ -4,9 +4,7 @@
 source 'https://rubygems.org'
 
 gem 'puppet', ENV.key?('PUPPET_VERSION') ? "~> #{ENV['PUPPET_VERSION']}" : '>= 5.5'
-
-# https://github.com/voxpupuli/puppet-selinux/issues/318
-gem 'facter', '< 4'
+gem 'facter', github: 'puppetlabs/facter-ng', branch: 'experimental_exit'
```

Comment:
Using the different branch is temporary, yes? As the issue will be resolved soon in the next facter release.